core/src/main/java/org/vertexium/GraphBase.java

```diff
@@ -913,4 +914,13 @@ public void ensurePropertyDefined(String name, Object value) {
         }
         return elements;
     }
+
+    @Override
+    public List<InputStream> getStreamingPropertyValueInputStreams(List<StreamingPropertyValue> streamingPropertyValues) {
+        List<InputStream> results = new ArrayList<>();
+        for (StreamingPropertyValue streamingPropertyValue : streamingPropertyValues) {
+            results.add(streamingPropertyValue.getInputStream());
+        }
+        return results;
+    }
```

Comment:
This might be a little cleaner with streams.
```
        return streamingPropertyValues
                .stream()
                .map(StreamingPropertyValue::getInputStream)
                .collect(Collectors.toList());
```